frontend/projects/upgrade/src/app/core/error-handler/app-error-handler.service.spec.ts

```diff
@@ -15,7 +15,7 @@ describe('AppErrorHandler', () => {
     service = new AppErrorHandler(mockNotificationsService, mockEnvironment);
   });
 
-  it('should call notification service with an error of "An error occured. See console for details." when not in production and not 401', () => {
+  it('should call notification service with an error of "An error occurred. See console for details." when not in production and not 401', () => {
```

Comment:
This test case needs to be modified removing the constraints of the environment. Also update the description of the test case